gateway/stateful-gateway/README.adoc

```diff
@@ -0,0 +1,23 @@
+= Stateless Gateway
+
+== Set-up
+
+The basic set-up we are going for is:
+
+* gateway at localhost:8080
+* dummy service at localhost:8081
+* keycloak at localhost:8180
+
+run `docker-compose up -d` to start a local keycloak server on port 8180
+
+* create a realm called `service` 
+* a client called `gateway` with secret `42aad01b-414e-4a80-8e6d-7b9332b3aebe` (or change the secret in the application.yml) 
```

Comment:
Het zal misschien aan mij liggen, maar als ik een `openid-connect` client aanmaak, dan zie ik geen optie om een secret in te geven.. Kun je deze instructies iets uitbreiden? Kan me voorstellen dat je ook bepaalde URLs moet ingeven bijvoorbeeld.

Reply:
ik denk dat het neerkomt op het aanpassen van de secret in de application.yml, ik zal dit ff checken/aanpassen.